services/blockchain-connector/shared/sdk/client.js

```diff
@@ -75,7 +75,8 @@ const checkIsClientAlive = async () =>
 		wsInstance.ping(() => {});
 
 		// eslint-disable-next-line consistent-return
-		setTimeout(() => {
+		const timeout = setTimeout(() => {
+			clearTimeout(timeout);
```

Comment:
Do we need to clear timeout here? The timeout would invalid now as callback function is already executed after timing out.

Reply:
It's unnecessary but a redundant measure we added based on my discussion with Ishan.